packages/default/scss/pivotgrid/_theme.scss

```diff
@@ -65,6 +65,9 @@
     .k-pivotgrid-cell.k-state-focus,
     .k-pivotgrid-cell.k-focus,
     .k-pivotgrid-cell:focus,
+    .k-pivotgrid-empty-cell.k-state-focus,
```

Comment:
let's delete the k-state-focus as we're about to remove all state selectors in the following PR: https://github.com/telerik/kendo-themes/pull/3877